.nvmrc

```
@@ -1 +1 @@
->=18.16.0
+lts/hydrogen
```

```suggestion
v18.17
```
Though we'll loose flexibility, it's better to use a fixed version than have floating versions which we may not know will work well. Also this will save us from multiple node versions saturating the disk.

---

Just adding a +1 for pinning to a concrete `major.minor.patch` version we know is compatible. Does the `v` prefix get picked up correctly from [shell integrations](https://github.com/nvm-sh/nvm/blob/master/README.md#L557)?

---

Looks fine to me:

```bash
cd content
Found 'path/to/content/.nvmrc' with version <v18.17>
Now using node v18.17.1 (npm v9.6.7)
```